sources/Cargo.toml

```diff
@@ -210,23 +210,23 @@ x509-parser = "0.16"
 base64 = "0.22"
 
 [workspace.dependencies.bottlerocket-modeled-types]
-git = "https://github.com/bottlerocket-os/bottlerocket-settings-sdk"
-tag = "bottlerocket-settings-models-v0.6.0"
+git = "https://github.com/vigh-m/bottlerocket-settings-sdk"
+tag = "test-1"
 version = "0.6.0"
 
 [workspace.dependencies.bottlerocket-settings-models]
-git = "https://github.com/bottlerocket-os/bottlerocket-settings-sdk"
-tag = "bottlerocket-settings-models-v0.6.0"
+git = "https://github.com/vigh-m/bottlerocket-settings-sdk"
+tag = "test-1"
 version = "0.6.0"
 
 [workspace.dependencies.bottlerocket-settings-plugin]
-git = "https://github.com/bottlerocket-os/bottlerocket-settings-sdk"
-tag = "bottlerocket-settings-plugin-v0.1.0"
+git = "https://github.com/vigh-m/bottlerocket-settings-sdk"
+tag = "test-1"
 version = "0.1.0"
 
 [workspace.dependencies.settings-extension-oci-defaults]
-git = "https://github.com/bottlerocket-os/bottlerocket-settings-sdk"
-tag = "bottlerocket-settings-models-v0.6.0"
+git = "https://github.com/vigh-m/bottlerocket-settings-sdk"
+tag = "test-1"
```

Comment:
Looks like we have not yet merged the sdk PR. Could you change the status of this core-kit PR to draft?

Reply:
Ack. I had these changes in a separate commit to make it easy to remove them later. I'll move to a draft as well.